CHANGELOG.md

```diff
@@ -5,23 +5,63 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- Stabilize `call_runtime` ‒ [#1749](https://github.com/paritytech/ink/pull/1749)
-- Make E2E testcases generic over `E2EBackend` trait - [#1867](https://github.com/paritytech/ink/pull/1867)
-- Modify static buffer size via environmental variables - [#1869](https://github.com/paritytech/ink/pull/1869)
-- Persist static buffer size in metadata - [#1880](https://github.com/paritytech/ink/pull/1880)
-- Add backend choice to the E2E testcase configuration ‒ [#1864](https://github.com/paritytech/ink/pull/1864)
 
-### Changed
-- Reexport `scale` dependencies, introduce `#[ink::scale_derive]` ‒ [#1890](https://github.com/paritytech/ink/pull/1890)
+## Version 5.0.0-alpha
+
+The preview release of the ink! 5.0.0 release.
+This release addresses the majority of issues raised in the OpenZeppelin audit
+in particular we addressed the proxy selector clashing attack.
+As of this release, ink! only allows exactly one other message with a well-known reserved selector to be defined.
+You can read more about the change in the [PR](https://github.com/paritytech/ink/pull/1708)
+
+There are also other notable changes:
+
+- Rework of event definitions.
```

Comment:
Link to PR here [#1827](https://github.com/paritytech/ink/pull/1810) I know it also below but saves looking.